addons/skin.estuary/xml/Variables.xml

```diff
@@ -329,7 +329,7 @@
 	<variable name="ListWatchedIconVar">
 		<value condition="ListItem.IsRecording">windows/pvr/record.png</value>
 		<value condition="ListItem.IsPlaying">overlays/watched/OverlayPlaying-List.png</value>
-		<value condition="ListItem.IsResumable">overlays/watched/resume.png</value>
+		<value condition="ListItem.IsResumable + Integer.IsEqual(ListItem.Playcount,0)">overlays/watched/resume.png</value>
```

Comment:
Maybe I dont understand correctly, but what happens if the playcount is >0 (in meaning i start watching a movie again for a 2nd time). AFAICS the resume flag will only be listed if the playcount equals to 0 which will not be the case if I watch a movie for the 2nd time, no?
So you won't see the resume flag never again, if I'm not mistaken.

Reply:
true, once watched.. you'll always see the watched icon.

Reply:
Hm, need to test that tomorrow. As a movie should also be shown as an 'in progress' movie even if I have watched it already. 

Not complaining, just not sure how it will work out ;)

Reply:
And not sure if that's important, but the default value for the advancedwettings.xml is already below 100%. It's set to 90% at advancedsettings.cpp as a default value